sam/template.yaml

```diff
@@ -0,0 +1,44 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  hallow
+  
+  Sample SAM Template for hallow
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 5
+
+Resources:
+  Hallow:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: ../
+      Handler: hallow
+      Runtime: go1.x
+      Tracing: Active # https://docs.aws.amazon.com/lambda/latest/dg/lambda-x-ray.html
+      Events:
+        CatchAll:
+          Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
+          Properties:
+            Path: /hello
```

Comment:
Eh?

Reply:
This is 90% the `sam` generated template file. Hilariously unused by us, so hasn't triggered an error yet. We can clean the whole thing up or just fix it when we need that particular bit of the template to look better. Thoughts?

Reply:
I think my preference is for stripping it down to just the pieces needed for us.